warehouse/query-core/src/main/java/datawave/query/jexl/visitors/FieldMissingFromSchemaVisitor.java

```diff
@@ -94,7 +94,7 @@ protected Object findMissingFields(JexlNode node, Object data) {
 
         for (ASTIdentifier identifier : identifiers) {
             String fieldName = JexlASTHelper.deconstructIdentifier(identifier);
-            if (!this.allFieldsForDatatypes.contains(fieldName) && !specialFields.contains(fieldName)) {
+            if ((!this.allFieldsForDatatypes.contains(fieldName) && !specialFields.contains(fieldName)) || helper.isHidden(fieldName, this.datatypeFilter)) {
```

Comment:
The documentation for the colf `h` states that it denotes an event that should be hidden from display.  Are you certain that it should be treated as non-existent within the context of the FieldMissingFromSchemaVisitor? This visitor is used in `DefaultQueryPlanner.timedTestForNonExistentFields()`, and will impact results there. If that's intended and correct, then disregard this comment.